Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -695,6 +695,10 @@
 #define ENABLE_LEGACY_PDFKIT_PLUGIN 1
 #endif
 
+#if !defined(ENABLE_UNIFIED_PDF) && PLATFORM(COCOA) && !PLATFORM(APPLETV) && !PLATFORM(WATCHOS)
```

Comment:
Seems OK (I am doing this through a local override already).

Reply:
Me too